### PR TITLE
Feature/store filter

### PR DESCRIPTION
**Pull request by RV-Adrian**
@ghislaineguerin This is as clean as I could get it. If you can remove log files, please feel free.
Can you merge when looking okay?  Thanks!

### CODE REVIEW

File: src/ui-components/filter.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        margin-right: 6px;
+        content: '✔';
+        background: url('') no-repeat;
+    }
+}
+
+
+/*MOBILE BREAKPOINT */
+
+@media screen and (max-width: 660px) {
```
Comment (ghislaineguerin) [inline src/ui-components/filter.scss]: @RV-Adrian are we adding new mobile breakpoints?
I think we stick to  768 - 922 - 1200 ?  320 - 480 Only when needed?

Comment (RV-Adrian) [inline src/ui-components/filter.scss]: @ghislaineguerin I've committed just now using the variables. (ie $screen-lg) Let me know if that works!

**Comment by ghislaineguerin** [comment]
@RV-Adrian why is this closed?

**Comment by RV-Adrian** [comment]
@ghislaineguerin Francisco helped me today with a rebase and getting everything up to date. All the changes in this branch are in now, so this PR should not be merged.  Unfortunately this one got a bit messy! Let's look at any issues separately today.

**Pull request closed without merging**